doc/chapters/api/index.rst

```diff
@@ -1,21 +1,10 @@
-
-.. _api:
-
-API
-***
-
-This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
-mostly useful for integrators.
-
-.. TODO::
-    This chapter is still to be written properly. Some bits and pieces have
-    been but they are under development.
-
```

Comment:
Why is all of this removed? The main heading and introduction are still valid, so just remove the TODO